python/Makefile.am

```diff
@@ -33,16 +33,28 @@ PY_LOG_COMPILER = $(PYTHON)
 AM_TESTS_ENVIRONMENT = PYTHONPATH='$(top_srcdir)/python/.libs'; export PYTHONPATH;
 
 if WITH_PYTHON
-  python_PYTHON = meep.py
-  pyexec_LTLIBRARIES = _meep.la
+  pkgpython_PYTHON = geom.py __init__.py
+  pkgpyexec_LTLIBRARIES = _meep.la
 endif
 
 if MAINTAINER_MODE
 
-meep-python.cpp:	$(SWIG_SRC) $(HPPFILES)
+meep-python.cpp: $(SWIG_SRC) $(HPPFILES)
 	swig -Wextra $(AM_CPPFLAGS) -outdir $(builddir) -c++ -python -o $@ $(srcdir)/meep.i
 
-meep.py:       		meep-python.cpp
+meep.py: meep-python.cpp
+
+__init__.py: meep.py
+	cp $< $@
+
+pymeep: .libs/_meep.so __init__.py
+	test -d meep || mkdir meep
+	cp __init__.py geom.py .libs/_meep.so meep
```

Comment:
Probably you need some `$(srcdir)` and `$(builddir)`s in here.

This is to support out-of-tree builds, and it would be a good idea to update travis so that we test this.  The way you do it is:
```
cd meep
mkdir build
cd build
../configure ...options...
make && make check && make install
```
instead of configuring and making in the top-level source directory.